### PR TITLE
Building with backgrounds and gradients

### PR DESCRIPTION
**Pull request by abruzy**

### CODE REVIEW

**Review by uche-inyama — COMMENTED** [review]
## Pull Request Under review by: Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi abruzy! you did a great job! :clap: 
### But there are changes you need to implement.

- [ ] The search tag is not positioned properly :point_down:  see snap shot below

Screenshot from 2019-09-03 21-37-13


- [ ] Add the ```<nav></nav>``` tag and ```<main></main> ``` tag in your html - writing a semantic html :smile:

- [ ] The images inside this class ```img-thumbnail``` should be wrapped with
   ```<a></a>``` tags and not  ```<div></div>```

### Implement these changes, and submit another code review form

### Cheers Henry. :smile:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                        </svg>
+                    </a></li>
+                    <li><a href="#news">Store</a></li>
+                    <li><a href="#contact">Mac</a></li>
+                    <li><a href="#about">iPod</a></li>
+                    <li><a href="#about">iPhone</a></li>
+                    <li><a href="#about">iPad</a></li>
+                    <li><a href="#about">iTunes</a></li>
+                    <li><a href="#about">Support</a></li>
+                    <li><a href="#">Search</a></li>
```
Comment (samgaco) [inline index.html]: **[REQUIRED]** This should have a search field. Your project should have a [search field](https://www.w3schools.com/howto/howto_css_searchbar.asp) on the rightmost section.

**Review by samgaco — CHANGES_REQUESTED** [review]
#### HI @abruzy :hand: 

#### Everything is looking neat :100: We still have some things to implement:

1. You should add a search field (more information below)
2. Stickler is not active on this repo yet. You have more information about how to implement it [here](https://github.com/microverseinc/linters-config/tree/master/css). Once you have it working, you should correct the errors if they come up.

### So, please just work on these 2 :crossed_fingers: points and submit back for another code review. Regards!
---------
* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. However, changing it now would take too much time. Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 
* [For your next code review requests]  Use meaningful names for their Pull Requests
* [For your next code review requests]  add a short summary of changes in Pull Request description

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review, by: Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi @abruzy , you have done well, implementing the changes :raised_hands: 
### But there's one change to implement
- [ ] The search field is still not properly positioned, it should be on the same line with the navigation items.
Kindly implement this change. Use these snapshots as a reference. :muscle: 
### Image of your clone:

Screenshot from 2019-09-17 00-18-51

### Image from [apple's site](www.apple.com)

Screenshot from 2019-09-17 00-26-21

### After aligning the search link properly, submit another form for code review

Your project hasn't been approved :stop_sign:

**Review by stratospherique — APPROVED** [review]
### Great work :clap: . You made all the necessary changes :100: . You project is approved :+1: :+1: and you can merge to master :smile: .

**Pull request merged**